docs/installing_agones.md

```diff
@@ -17,6 +17,8 @@ In this quickstart, we will create a Kubernetes cluster, and populate it with th
    1. [Starting Minikube](#starting-minikube)
```

Comment:
Thought: should `docs/installing_agones.md` be the `README.md` in `/install` ?

Reply:
i don't think it's good idea move this file to `install` directory. With the new directory structure proposed above, it would not be necessary, but is my opinion, open to changes

Reply:
My thought is that (a) it would be contextual - i.e. the install folder has a README, and the content for the README, would be the contents of `docs/intaling_agones.md`

It will also (b) make release a tad cleaned - we can zip up the `install` dir and it has the docs, and the yaml and the chart and we are good to go for both pathways.

Reply:
I also think it's a good idea, though it would required to update the `docs/intaling_agones.md` to fit it new destination. Should we create a new issue for it ?

Reply:
I like this - let's create a separate issue for this, and track it in a seperate PR :smile: follow my own advice for smaller PRs